electron: 28.3.1 -> 28.3.3, 29.4.0 -> 29.4.2, 30.0.6 -> 30.1.1

yayayayaka opened the pull request:
## Description of changes

### Electron 28
- Changelog: https://github.com/electron/electron/releases/tag/v28.3.3
- Diff: https://github.com/electron/electron/compare/refs/tags/v28.3.1...v28.3.3

### Electron 29
- Changelog: https://github.com/electron/electron/releases/tag/v29.4.2
- Diff: https://github.com/electron/electron/compare/refs/tags/v29.4.0...v29.4.2

### Electron 30
- Changelog: https://github.com/electron/electron/releases/tag/v30.1.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v30.0.6...v30.1.1

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

teutat3s reviewed (approved):
LGTM, do you want to bump electron_30 to [30.1.0](https://github.com/electron/electron/releases/tag/v30.1.0) here as well?

yayayayaka commented:
Sure, can do.

yayayayaka committed:
electron_28-bin: 28.3.1 -> 28.3.3

- Changelog: https://github.com/electron/electron/releases/tag/v28.3.3
- Diff: https://github.com/electron/electron/compare/refs/tags/v28.3.1...v28.3.3

yayayayaka committed:
electron_29-bin: 29.4.0 -> 29.4.2

- Changelog: https://github.com/electron/electron/releases/tag/v29.4.2
- Diff: https://github.com/electron/electron/compare/refs/tags/v29.4.0...v29.4.2

yayayayaka committed:
electron_30-bin: 30.0.6 -> 30.1.1

- Changelog: https://github.com/electron/electron/releases/tag/v30.1.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v30.0.6...v30.1.1

yayayayaka committed:
electron-source.electron_29: 29.4.0 -> 29.4.2

- Changelog: https://github.com/electron/electron/releases/tag/v29.4.2
- Diff: https://github.com/electron/electron/compare/refs/tags/v29.4.0...v29.4.2

yayayayaka committed:
electron-source.electron_30: 30.0.6 -> 30.1.1

- Changelog: https://github.com/electron/electron/releases/tag/v30.1.1
- Diff: https://github.com/electron/electron/compare/refs/tags/v30.0.6...v30.1.1

emilylange commented:
Result of `nixpkgs-review pr 317044` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages marked as broken and skipped:</summary>
  <ul>
    <li>blockbench</li>
    <li>deltachat-desktop</li>
    <li>electron_28</li>
    <li>electron_28-bin</li>
    <li>itch</li>
    <li>lx-music-desktop</li>
    <li>mattermost-desktop</li>
  </ul>
</details>
<details>
  <summary>62 packages built:</summary>
  <ul>
    <li>affine</li>
    <li>antares</li>
    <li>bilibili</li>
    <li>bitwarden-desktop</li>
    <li>bitwarden-directory-connector</li>
    <li>breitbandmessung</li>
    <li>bruno</li>
    <li>camunda-modeler</li>
    <li>drawio</li>
    <li>drawio-headless</li>
    <li>electron (electron_30)</li>
    <li>electron-bin (electron_30-bin)</li>
    <li>electron_29</li>
    <li>electron_29-bin</li>
    <li>element-desktop</li>
    <li>element-desktop-wayland</li>
    <li>fcast-receiver</li>
    <li>feishin</li>
    <li>freetube</li>
    <li>headset</li>
    <li>heroic</li>
    <li>heroic-unwrapped</li>
    <li>jitsi-meet-electron</li>
    <li>koodo-reader</li>
    <li>kuro</li>
    <li>morgen</li>
    <li>nix-tour</li>
    <li>obsidian</li>
    <li>open-stage-control</li>
    <li>pandoc-drawio-filter</li>
    <li>pandoc-drawio-filter.dist</li>
    <li>passky-desktop</li>
    <li>podman-desktop</li>
    <li>pritunl-client</li>
    <li>proton-pass</li>
    <li>protonmail-desktop</li>
    <li>python311Packages.mkdocs-drawio-exporter</li>
    <li>python311Packages.mkdocs-drawio-exporter.dist</li>
    <li>python312Packages.mkdocs-drawio-exporter</li>
    <li>python312Packages.mkdocs-drawio-exporter.dist</li>
    <li>r2modman</li>
    <li>redisinsight</li>
    <li>revolt-desktop</li>
    <li>sharedown</li>
    <li>siyuan</li>
    <li>standardnotes</li>
    <li>stretchly</li>
    <li>super-productivity</li>
    <li>teams-for-linux</li>
    <li>terra-station</li>
    <li>tetrio-desktop</li>
    <li>thedesk</li>
    <li>threema-desktop</li>
    <li>uhk-agent</li>
    <li>uhk-udev-rules</li>
    <li>uivonim</li>
    <li>vesktop</li>
    <li>vieb</li>
    <li>webcord</li>
    <li>webcord-vencord</li>
    <li>youtube-music</li>
    <li>ytdownloader</li>
  </ul>
</details>

~~~
# git describe --abbrev=40
24.05-pre-87571-ga3ba07a2b6fb2d5374ccad78228b136ca8bd37da
~~~

`element-desktop` runs fine under wayland/sway.
